src/main/scala/fs2/ftp/UnsecureFtp.scala

```diff
@@ -72,19 +70,17 @@ final private class UnsecureFtp[F[_]: ConcurrentEffect: ContextShift](
         )
 
   def execute[T](f: JFTPClient => T): F[T] =
-    blocker.delay[F, T](f(unsafeClient))
+    Sync[F].delay(f(unsafeClient))
```

Comment:
Sync[F].delay is not equivalent to blocker.delay

please use `Sync[F].blocking()`